components/core/OnboardingModal.js

```diff
@@ -1,9 +1,11 @@
 import * as React from "react";
 import * as Constants from "~/common/constants";
+import * as Actions from "~/common/actions";
 
 import { css } from "@emotion/react";
-import { ButtonPrimary } from "~/components/system/components/Buttons";
+import { ButtonPrimary, ButtonSeconddary } from "~/components/system/components/Buttons";
```

Comment:
Actually, just small typo here, but I can fix that

Reply:
nice catch! thanks